### ISSUE TITLE
Scale (from center point)

### ISSUE THREAD

**Issue by Midhil457** [4ian/GDevelop#2827]
Currently, if you use the scale action, the object's scale is changed from the `origin` and there is no option to scale it from center. It is a really useful action and will make things way easier. What I currently do is use a tween with `scale from center` enabled with a low duration, which doesn't work most of the time

## Description

Currently, if you scale an object, it scaled from the `origin` which is a bit frustrating as it changes the `center` of the object to somewhere else and break all other events that use the center point of an object. For example, I am setting the scale of an object to 0, and then using tween to smoothly scale it up from the center (`scale from center enabled`) for a transition, but as the `scale` action changed the center point, now the object is tweened with an offset.

Here is an example, instead of the `PAUSED` text being in the center top of the big circle, it gets offset because of the `scale` action
![centeroffset](https://user-images.githubusercontent.com/73597906/126448161-577d4706-92fe-4ade-bbf9-a4dda73fdc84.PNG)
Events:
![cnetroffsetevents](https://user-images.githubusercontent.com/73597906/126448231-99b05f66-ff4c-4c93-9a3e-7419539fef30.PNG)

## Solution suggested

Add an `scale from center` toggle just like the tween scale actions have

Explain if you can help to implement the solution.

**Comment by beanmatt** [comment]
@Midhil457 Have you tried moving the origin point in the Object editor?  This is what I did to have then scale from the center.  If I need to reference that Object's "normal" X/Y position in other events, I just do `Object.X() - (Object.Width()/2)` or `Object.Y() - Object.Height()/2`.

**Comment by Midhil457** [comment]
Yes, that possible for sprites, but sadly its not possible to change the points for other object types

:) Thanks
